lib/zuul.js

```diff
@@ -1,4 +1,5 @@
 var debug = require('debug')('zuul');
+var omit = require('lodash').omit;
```

Comment:
lodash is already in package.json so we better reuse it than install a new 'lodash.omit' dependency right?